### ISSUE TITLE
"Error: Network Error"

### ISSUE THREAD

**Issue by Apapora** [aws-samples/aws-bookstore-demo-app#55]
**Describe the bug**
Getting a "Error: Network Error" pop up message when loading the "main" webpage after log in. Seems to be a CORS issue:
"Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://yuq7ws39pj.execute-api.us-east-1.amazonaws.com/prod/recommendations. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 502."

**To Reproduce**
1. Log in
2. Load the webpage (https://d2h3ljlsmzojxz.cloudfront.net/)

**Expected behavior**
Expected no error

**Screenshots**
![error](https://github.com/aws-samples/aws-bookstore-demo-app/assets/47843562/8f53133b-609e-4730-8e00-9ff32f5562eb)

**Desktop (please complete the following information):**
 - Firefox

**Comment by triggan** [comment]
Yes, that is the old demo stack that I need to deprecate.  I'll be updating the README soon with the latest stack deployed here:  https://d2n8xfsn53tzfm.cloudfront.net/

Let me know if you have any issues with this other stack.
